app/helpers/series_helper.rb

```diff
@@ -58,13 +58,14 @@ def work_series_description(work, series)
     ts("Part <strong>%{position}</strong> of %{title}".html_safe, position: serial.position, title: link_to(series.title, series))
   end
 
-  def series_list_for_feeds(work)
+  def series_list_with_work_position(work, email_styling: false)
     series = work.series
-    list = []
-    series.each do |s|
-      list << ts("Part %{serial_index} of %{link_to_series}", serial_index: s.serial_works.where(work_id: work.id).select(:position).first.position, link_to_series: link_to(s.title, series_url(s)))
+    return t("series_helper.none") if series.empty?
+
+    list = series.map do |s|
+      t("series_helper.series_description_html", index: s.serial_works.where(work_id: work.id).select(:position).first.position, series_link: email_styling ? style_link(s.title, series_url(s)) : link_to(s.title, series_url(s)))
     end
-    list.join(", ")
+    list.join(t("support.array.words_connector"))
```

Comment:
Does this work with just a `join`? I would have expected it to need a [safe_join](https://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-safe_join) instead to preserve the HTML safe status. Maybe we could get rid of the html_safe calls at the call sites if we switch to `safe_join`?